src/argv.ts

```diff
@@ -254,8 +256,9 @@ export class Argv {
         return this.map.get("containerExecutable") ?? "docker";
     }
 
-    get enableJsonSchemaValidation (): boolean {
-        return this.map.get("enableJsonSchemaValidation") ?? true;
+    get jsonSchemaValidation (): boolean {
```

Comment:
This might be breaking changes for people that's using `enableJsonSchemaValidation`.

We might want to emphasis this in the release note of this breaking changes in this "beta" flag

also, agree that `enable` is redundant here

Reply:
Yes, it's acceptable, since the flag is so new.